client/src/App.css

```diff
@@ -128,33 +100,26 @@ nav a.active {
     box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
 }
 
-
-
 /* buttons */
 
 .login-signin-buttons {
  display: flex;
  justify-content: center;
  gap: 20px;
-
 }
 
-
 button {
     padding: 15px 25px;
     background-color: rgb(219, 104, 104);
     border: none;
-    border-radius: 7px;
-    display: block;
+    border-radius: 7px;   
     width: 110px;
-    margin: 20px auto;
     text-decoration: none;
     color: rgb(251, 249, 249);
     font-family: "Montserrat";
     font-weight: 700;
     font-size: 0.9rem;
     text-align: center;
-
 }
 
 .login-signin-buttons button :hover {
```

Comment:
1. What is selected with this hover rule? What is inside this button?

2. What happens when someone focuses this button without hovering?

https://css-tricks.com/keyboard-only-focus-styles/